PluginCore/PluginCore/Helpers/PathHelper.cs

```diff
@@ -19,7 +20,7 @@ public static String BaseDir
         {
             get
             {
-                if (PluginBase.MainForm.StandaloneMode) return AppDir;
+                if (PluginBase.MainForm.StandaloneMode) return Path.GetDirectoryName(GetAssemblyPath(Assembly.GetExecutingAssembly()));
```

Comment:
Doesn't this make `BaseDir` possibly return a different path from `AppDir` in standalone mode?
Shouldn't `AppDir` be changed accordingly, and `BaseDir` still return `AppDir`?